project/JavalibIRCleaner.scala

```diff
@@ -262,6 +262,33 @@ final class JavalibIRCleaner(baseDirectoryURI: URI) {
               JSUnaryOp(JSUnaryOp.!, JSUnaryOp(JSUnaryOp.!, arg)),
               BooleanType)
 
+        // 2.11 s"..." interpolator
```

Comment:
Just to check my understanding: We can remove this once the javalib is independent of the Scalalib, right?

Reply:
Yes, that's correct. Or to be more precise: once it is independent, we can publish a single `javalib`, compiled with a single Scala compiler version (say 2.12), so we can remove the 2.11 build of the javalib and hence remove this rewrite.